ring/spread_minimizing_token_generator_test.go

```diff
@@ -458,7 +458,14 @@ func TestSpreadMinimizingTokenGenerator_GetMissingTokens(t *testing.T) {
 	// we get all the tokens for the underlying instance, but we don't mark all of them as taken
 	// in order to simulate that some tokens were taken by another instance when the method was
 	// first called
-	missingIndexes := []int{rand.Intn(tokensPerInstance - 1), rand.Intn(tokensPerInstance - 1), rand.Intn(tokensPerInstance - 1)}
+	missingIndexes := make([]int, 0, 3)
+	for i := 0; i < 3; i++ {
+		missingIndex := rand.Intn(tokensPerInstance - 1)
+		if slices.Contains(missingIndexes, missingIndex) {
+			continue
```

Comment:
[nit] Might be good to add a comment explaining we need three unique values

Reply:
I have added a comment. Thank you.